### DISCUSSION TITLE
Customizing AttachAction Select

### DISCUSSION THREAD

**Discussion by gregoryloichot** [filamentphp/filament#13676] [Help] [answered]
### Package

Actions

### Package Version

v3.2

### How can we help you?

In a relation manager, I have on the table an header action to attach a new object 👍 

```php
…
->headerActions([
                Tables\Actions\AttachAction::make()
                    ->label('Add assistant')
                    ->recordSelectSearchColumns(['surname', 'name']),
            ])
…
```

For a reason I do not understand, it works well and generate a Select in a modal, but : the options "label" are the surname attribute of the object User. I want to customize this label to be `$user->name .' '. $user->surname`.

So, my questions are:

- How do I modify the select entries to be labeled as above ?

I tried to add a form to my action, but I get stuck at some point.

Thanks for your help.

**Comment by leandrocfe** [comment, marked as answer]
try this
https://github.com/filamentphp/filament/discussions/13096#discussioncomment-9674355
